packages/extension-ui/src/Popup/Derive/SelectParent.tsx

```diff
@@ -221,21 +237,23 @@ function SelectParent({
             </Warning>
           )}
         </InputWrapper>
-      </ContentContainer>
+      </Form>
       <VerticalSpace />
       <ButtonArea footer={footer}>
         <Button
           isDisabled={isBusy}
           onClick={goTo(`/account/edit-menu/${parentAddress}?isExternal=${externalString}`)}
           secondary
+          type='button'
         >
           {t<string>('Cancel')}
         </Button>
         <Button
           data-button-action='create derived account'
```

Comment:
it's a correct `type` value for a button that serves as js callback trigger. Default `type` value - `submit` - should be used on buttons that... well, submit forms. I could probably override the default in `Button` component since all buttons in project should have this value :sweat_smile:

Reply:
I would leave it as it is, even though most of the time we want button type=button, I think overriding default `submit` could cause some confusion - I don't see much benefit tbh

Reply:
I didn't mean the `type='button'` - I meant the `data-button-action` in line 252. GH unfortunatelly doesn't properly highlight the selected line in a comment, if it's unmodified.

---

Good that you mentioned `type='button'` though - I admit I was curious about it :D Tbh I wasn't aware that the default value is not `button`, although seems that it's not that simple either, depending on where a particular button is located in the DOM (if I understand the specs correctly):
> [`'submit'`] is the default if the [`'type'`] attribute is not specified for buttons associated with a `<form>`, or if the attribute is an empty or invalid value.

_(source: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button)_

I would generally opt for making the `button` value the default one, cause I think `<button>`s are used in much more cases than just as a form submit button (providing an app is built DOM-semantically), but in this particular case I don't mind not doing that at all.